raiden/tests/scenarios/pfs7_simple_path_rewards.yaml

```diff
@@ -68,7 +68,7 @@ scenario:
             - wait: 10
             - assert_pfs_history: {source: 3, target: 0, request_count: 2}
             - assert_pfs_iou: {source: 3, amount: 200}
-            - assert_pfs_iou: {source: 2, amount: 100}
+            - assert_pfs_iou: {source: 2, amount: 0}
             - assert_pfs_iou: {source: 1, amount: 0}
```

Comment:
same here, maybe just do not assert on an no-existing iou. And if you like to to do that, there is the `iou_exists: False` assertion on that task in the scenario player

Reply:
Yeah, I think the check should still be there, since it checks that the mediating nodes indeed aren't paying anything to the PFS. But I think changing it to `iou_exists: False` is a more verbose solution so will update to that

Reply:
nice, thank you! Is it `False` or does `false` also work? Did you run that locally?

Reply:
It's `false`. That's what we use in other places at least.